Dockerfile

```diff
@@ -13,7 +13,7 @@ ENV GROUP_NAME=app
 ENV HOME=/app
 ENV GROUP_ID=11000
 ENV USER_ID=11001
-ENV PORT=8080
```

Comment:
skal være 8080 når den deploys. Brug dockerfile.dev til dev ting

Reply:
@st-randers

Reply:
Modtaget, bare sært at frontend udstilles på port 8080 her, og 3000 ellers.

Reply:
```npm run dev``` laver en server og vite config filen sætter porten til 3000.
Men når det bygges med dockerfile til deployment, så er python/flask serveren og vue filerne bliver serveret her: https://github.com/Randers-Kommune-Digitalisering/vue-python-template/blob/develop/flask/src/main.py#L31-L37

Så der er kun én port. Men i dev er der to servere og defor to porte. Vite serveren tjekker efter fil-ændringer for vue filer og flask serveren er sat til debug i docker-compose og tjekker efter fil-ændringer for flask/python.

Håber det giver lidt mening 😅